app/assets/stylesheets/pages/_main.scss

```diff
@@ -0,0 +1,110 @@
+.header-container {
+	@extend .container;
+	padding: 1em;
+
+	@media (min-width: $largish) {
+		background-position: top right;
+		padding: 3em;
+	}
+}
+
+.main-title {
+	@extend .main-title-layout;
+	text-align: center;
+	color: $white;
+	padding-top: 1em;
+	padding-bottom: 1em;
+	border: 2px solid white;
+	border-radius: 2px;
+	transition: all 0.2s ease 0s;
+	font-size: 1.3em;
+
+	@media (min-width: $mediumish) {
+		font-size: 1.6em;
+		border: 3px solid white;
+	}
+
+	@media (min-width: $largish) {
+		font-size: 1.9em;
+	}
+}
+
+a.logo { 
+	text-decoration: none;
+	&:hover {
+		text-decoration: none;
+	}
+}
+
+.logo-img {
```

Comment:
@Tallys Should this be nested under `a.logo`? Could it even be just `a.logo img`?